server/src/entity/Questions.ts

```diff
@@ -0,0 +1,30 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({name: 'qms_question'})
+export class Questions {
+    @PrimaryGeneratedColumn('uuid', { name: 'id' }) id: string;
+
+    @Column('varchar', {
+        name: 'type',
+        nullable: false,
+    })
+    type: string;
+
+    @Column('varchar', {
+        name: 'text',
+        nullable: false,
+    })
+    text: string;
+
+    @Column('varchar', {
+        name: 'answer',
```

Comment:
```suggestion
        name: 'correct_answer',
```
As there is going to be a class `AnsweredQuestions` extending this one with a field `competitor_answer` / `obtained_answer`